README.md

```diff
@@ -34,8 +34,7 @@ We're not sure which route you might have taken on your way here, but we're real
 
 - [Eclipse OpenJ9 website](http://www.eclipse.org/openj9) - Learn about this high performance, enterprise-grade Java Virtual Machine (JVM) and why we think you want to get involved in its development.
 - [AdoptOpenJDK website](https://adoptopenjdk.net/releases.html?variant=openjdk9-openj9) - Grab pre-built OpenJDK binaries that embed OpenJ9 and try it out.
-- [Build instructions](https://www.eclipse.org/openj9/oj9_build.html) - Here's how you can build an OpenJDK with OpenJ9 yourself.
-- [FAQ](https://www.eclipse.org/openj9/oj9_faq.html)
+- Build instructions for [JDK8](https://github.com/eclipse/openj9/blob/master/doc/build-instructions/Build_Instructions_V8.md), [JDK11](https://github.com/eclipse/openj9/blob/master/doc/build-instructions/Build_Instructions_V11.md), and [JDK14](https://github.com/eclipse/openj9/blob/master/doc/build-instructions/Build_Instructions_V14.md) - Here's how you can build an OpenJDK with OpenJ9 yourself.
```

Comment:
I'd prefer to link to the directory instead of jdk14 directly, so we don't need to update this every 6 months.
https://github.com/eclipse/openj9/blob/master/doc/build-instructions

Reply:
I've replaced the JDK14 link with an "and More" link to the directory. Let me know if this works for you.